src/pcf_flags.cpp

```diff
@@ -0,0 +1,107 @@
+/**
+ * @file pcf_flags.cpp
+ *
+ * @author  Guy Maurel
+ * @license GPL v2+
+ */
+
+#define DEFINE_PCF_NAMES
```

Comment:
Does this (and the corresponding `#ifdef` below, which will always be true?) still serve a purpose?

Reply:
no, thanks